### DISCUSSION TITLE
采用v2ray+ws+tsl 端口4534 . 不到一个小时就被封了，域名是在腾讯云上解析，vps采用美国的垃圾供应商

### DISCUSSION THREAD

**Discussion by tedyang1234** [v2fly/v2ray-core#1890] [General]
采用v2ray+ws+tsl 端口4534 . 不到一个小时就被封了，域名是在腾讯云上解析，vps采用美国的垃圾供应商
是端口的问题吗 ？我改成443

如果你遇到的问题不是 V2Ray 的 bug，比如你不清楚要如何配置，请使用[Discussion](https://github.com/v2fly/v2ray-core/discussions)进行讨论。

此 Issue 会被立即关闭。

If you are not sure if your question is truely a bug in V2Ray, please discuss it [here](https://github.com/v2fly/v2ray-core/discussions) first.

This issue will be closed immediately.

**Comment by simplerick-simplefun** [comment]
端口443，放一个网页caddy/nginx分流。妥妥的。

**Comment by chika0801** [comment]
你搭好后，在VPS配置上，添加了禁止VPS访问CN IP的规则没有。

如果没有，你是否（有意无意的）比如手机上面，使用了全局代理，你不太懂时，WX，WB等通过代理访问了国内。如果有请改一下。

被封的是那个端口吧？不是你VPS IP被封了？

**Comment by tedyang1234** [comment]
> 

手机的v2ray应该是没有全局，我现在已经放弃了 ，直接用cdn，虽然很慢，但是上个外网就可以了。也不指望看油管了

**Comment by chika0801** [comment]
我建议你不要爱惜VPS IP，何必套CDN天天让自己减速了2333

**Comment by tedyang1234** [comment]
> 我建议你不要爱惜VPS IP，何必套CDN天天让自己减速了2333

我用的是最垃圾的vps.  才9刀好像，我查了下我隔壁的一个IP被google封了应该是影响到我了 ，所以很容易被封，今天刚被放出来。cdn虽然减速了，但是再也不担心被失联了，懒得配置，为了连上我被ban的vps我到处找免费的节点，太浪费时间
